deploy/crds/spinnaker.io_spinnakeraccounts_crd.yaml

```
@@ -1,460 +1,456 @@
+                                token:
+                                  description: Token is the bearer token for authentication
+                                    to the kubernetes cluster.
+                                  type: string
+                                tokenFile:
+                                  description: TokenFile is a pointer to a file that
+                                    contains a bearer token (as described above).  If
+                                    both Token and TokenFile are present, Token takes
+                                    precedence.
+                                  type: string
+                                username:
+                                  description: Username is the username for basic authentication
+                                    to the kubernetes cluster.
+                                  type: string
+                              type: object
+                          required:
+                            - name
+                            - user
+                          type: object
+                        type: array
+                    required:
+                      - clusters
+                      - contexts
+                      - current-context
+                      - preferences
+                      - users
+                    type: object
+                  kubeconfigFile:
+                    description: KubeconfigFile referenced as an encrypted secret
+                    type: string
+                  kubeconfigSecret:
+                    description: Kubeconfig referenced as a Kubernetes secret
+                    properties:
+                      key:
+                        type: string
+                      name:
+                        type: string
+                    required:
+                      - key
+                      - name
+                    type: object
+                  useServiceAccount:
+                    description: UseServiceAccount authenticate to the target cluster
+                      using the service account mounted in Spinnaker's pods
+                    type: boolean
+                type: object
+              permissions:
+                additionalProperties:
+                  items:
+                    type: string
+                  type: array
+                type: object
+              settings:
+                type: string
+              validation:
+                properties:
+                  enabled:
+                    description: Enable or disable validation, defaults to false
+                    type: boolean
+                  failOnError:
+                    description: Report errors but do not fail validation, defaults
+                      to true
+                    type: boolean
+                  frequencySeconds:
+                    x-kubernetes-int-or-string: true
+                    description: Number of seconds between each validation
```

I found that you can still use the `anyOf` syntax, but the order of the types matters. 
In the old version it was 
```
anyOf:
  - type: string
  - type: integer
```  
but 
```
anyOf:
  - type: integer
  - type: string
```  
avoids the need to use `x-kubernetes-int-or-string`. My interpretation of `x-kubernetes-int-or-string` was a workaround, but that may be misguided.

```suggestion
                  frequencySeconds:
                    type: object
                    anyOf:
                      - type: integer
                      - type: string
                    description: Number of seconds between each validation
```

---

The above suggestion can be applied to all `int-or-string` lines.

---

Reading more of the documentation looks like `x-kubernetes-int-or-string` is just shorthand, not necessarily a workaround. Carry on!